lib/database_config.rb

```diff
@@ -14,7 +22,7 @@ def adapter
         'postgresql'
       end
 
-      unless SUPPORTED_ADAPTERS.include?(adapter)
+      unless DatabaseConfig::SUPPORTED_ADAPTERS.include?(adapter)
```

Comment:
??? I don't care really, but I'm curious, did this not work without fully qualifying or something?

Reply:
I think I just made a mistake with the refactor :|